styles/content/typography/body/styles.less

```diff
@@ -2,45 +2,40 @@
   /* body */
 
   html,
-  body,
-  p {
+  body {
     .element-text-size(body, null);
     .element-text-style(body);
     display: block;
 
     & when (@body-emphasis-enabled) {
 
-      &.emphasis,
-      .emphasis {
+      span.emphasis {
```

Comment:
No, I'm definitely in favor of the generalized class. But this line is adding a `span` selector (so the full selector is `span.emphasis`), so it's more specific.